### PR TITLE
refactor enable/disable text logic to partial and controller from model

### PR DESCRIPTION
**Pull request by abwmn**
DRY it up and relocate formatting responsibility to view/controller

### CODE REVIEW

File: app/views/admin/merchants/_merchant_partial.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="merchant<%= merchant.id %>">
```
Comment (thawley2) [inline app/views/admin/merchants/_merchant_partial.html.erb:1]: This is great, I think Nick and I will steal this :)

File: app/views/admin/merchants/index.html.erb [hunk 2]
```diff
@@ -2,18 +2,12 @@
 
 <div class="enabled_merchants">
   <% @merchants.enabled.each do |merchant| %>
-    <div class="merchant<%= merchant.id%>">
-      <%= link_to merchant.name, admin_merchant_path(merchant) %><br>
-      <%= button_to merchant.button_text, admin_merchant_path(merchant, switch_enabled: true), method: :patch %>
-    </div>
+    <%= render partial: 'merchant_partial', locals: { merchant: merchant } %>
```
Comment (thawley2) [inline app/views/admin/merchants/index.html.erb:5]: Great refactor!

**Review by thawley2 — APPROVED** [review]
This refactor looks great!! Nicely done.

**Review by sicknacco — APPROVED** [review]
Nice refactor!
I like usage of `?`

**Pull request merged**